starter-code/database.js

```diff
@@ -57,8 +57,10 @@ class Database {
         //  LAB 2
         // Implement the query to insert a user
         // remeber once it's finish to comment callback('Error listing users');
-        
-        callback('Error listing users');
+        database.collection("users").find().toArray((error, users)=> {
```

Comment:
You can use the callback directly as in the previous question, like this: `.toArray(callback)`.